### PR TITLE
UI for passthrough-related scene locking and resetting

### PR DESCRIPTION
**Pull request by andybak**

### CODE REVIEW

File: Assets/Scripts/GUI/LightingPopUpWindow.cs [hunk 1]
```diff
@@ -58,21 +67,25 @@ override public void Init(GameObject rParent, string sText)
...
-                if (env.m_Guid.ToString() == "e38af599-4575-46ff-a040-459703dbcd36")
+                if (env.m_Guid.ToString() == PASSTHROUGH_GUID)
                 {
                     m_Environments.Remove(env);
                     break;
                 }
             }
 #endif // PASSTHROUGH_SUPPORTED
 
+            // TODO for debugging only
```
Comment (mikeage) [inline Assets/Scripts/GUI/LightingPopUpWindow.cs:78]: Is this supposed to remain?

Comment (andybak) [reply Assets/Scripts/GUI/LightingPopUpWindow.cs]: fixed

**Review by mikeage — COMMENTED** [review]
looks good to me on a code level but I didn't test

**Comment by andybak** [comment]
I'd love to merge this before the code freeze but it would be good to get another pair of eyes on this (running it ideally).

@mikeskydev  ?

**Pull request merged**